From 127b7bb55cf919d611f2e176db0c119c38530e9d Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Tue, 3 Nov 2015 19:58:39 +0100
Subject: [PATCH 1/3] fix gcc 5

---
 src/core/AutomatableModel.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/core/AutomatableModel.cpp b/src/core/AutomatableModel.cpp
index 0ce80c4..ba9e945 100644
--- a/src/core/AutomatableModel.cpp
+++ b/src/core/AutomatableModel.cpp
@@ -312,8 +312,8 @@ void AutomatableModel::setAutomatedValue( const float value )
 									it != m_linkedModels.end(); ++it )
 		{
 			if( (*it)->m_setValueDepth < 1 &&
-				!(*it)->fittedValue( m_value ) !=
-							 (*it)->m_value )
+			    !((*it)->fittedValue( m_value ) !=
+			      (*it)->m_value) )
 			{
 				(*it)->setAutomatedValue( value );
 			}
-- 
2.4.3

