From d8c89c7e6b00471a8dd535cfc60081fd850d7c88 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Tue, 3 Nov 2015 20:01:37 +0100
Subject: [PATCH 3/3] fix lib64 carla

---
 plugins/carlabase/carla.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/carlabase/carla.cpp b/plugins/carlabase/carla.cpp
index f512ad2..553701e 100644
--- a/plugins/carlabase/carla.cpp
+++ b/plugins/carlabase/carla.cpp
@@ -153,7 +153,7 @@ CarlaInstrument::CarlaInstrument(InstrumentTrack* const instrumentTrack, const D
     QString dllName(carla_get_library_filename());
 
 #if defined(CARLA_OS_LINUX)
-    fHost.resourceDir = strdup(QString(dllName.split("/lib/carla")[0] + "/share/carla/resources/").toUtf8().constData());
+    fHost.resourceDir = strdup(QString(dllName.split("/lib64/carla")[0] + "/share/carla/resources/").toUtf8().constData());
 #else
     fHost.resourceDir = NULL;
 #endif
-- 
2.4.3

