From cc287628881cc150970e473792e2a2e7eb042287 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sat, 30 May 2015 13:49:15 +0200
Subject: [PATCH] Build with vst support but without having wine.

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 63af96c..9871ac1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -321,6 +321,9 @@ IF(LMMS_BUILD_WIN32)
 	SET(LMMS_SUPPORT_VST TRUE)
 	SET(STATUS_VST "OK")
 ENDIF(LMMS_BUILD_WIN32)
+IF(WANT_VST_NOWINE)
+	SET(LMMS_SUPPORT_VST TRUE)
+ENDIF(WANT_VST_NOWINE)
 
 
 # check for libsamplerate
-- 
2.1.0

