From a486817bb30e0b738fedc1cde34fd5117d4a796a Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Tue, 3 Nov 2015 20:00:08 +0100
Subject: [PATCH 2/3] fix case carla

---
 plugins/carlabase/carla.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plugins/carlabase/carla.cpp b/plugins/carlabase/carla.cpp
index db14178..f512ad2 100644
--- a/plugins/carlabase/carla.cpp
+++ b/plugins/carlabase/carla.cpp
@@ -258,6 +258,7 @@ intptr_t CarlaInstrument::handleDispatcher(const NativeHostDispatcherOpcode opco
     case NATIVE_HOST_OPCODE_RELOAD_PARAMETERS:
     case NATIVE_HOST_OPCODE_RELOAD_MIDI_PROGRAMS:
     case NATIVE_HOST_OPCODE_RELOAD_ALL:
+    case NATIVE_HOST_OPCODE_HOST_IDLE:
         // nothing
         break;
     case NATIVE_HOST_OPCODE_UI_UNAVAILABLE:
-- 
2.4.3

