From c79ae765851fed30fc8578ab960ea61873eff389 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Wed, 16 Mar 2016 16:08:21 +0100
Subject: [PATCH] fix install path

---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 88ed301..5d1ab51 100644
--- a/Makefile
+++ b/Makefile
@@ -1,15 +1,15 @@
 all: sfarkxtc
 
+BIN_PATH?=/usr/local/bin
 INSTALL?=install
 
 # Seems to be needed as clang doesn't appear to add this by default on
 # OSX. Let me know if you know a better way.
-LDFLAGS=-L/usr/local/lib
-
+LDFLAGS=-I../sfArkLib -L../sfArkLib
 LDLIBS=-lsfark -lz
 
 clean: 
 	-rm sfarkxtc 2>/dev/null || true
 
 install: sfarkxtc
-	$(INSTALL) -D sfarkxtc $(DESTDIR)/usr/local/bin/sfarkxtc
+	$(INSTALL) -D sfarkxtc $(DESTDIR)/${BIN_PATH}/sfarkxtc
-- 
2.5.0

