From cab0ee8bb4474335a7f465228ebdb9db893c4f3e Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sat, 2 Apr 2016 15:44:41 +0200
Subject: [PATCH] customize makefile

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f8cb40e..aa3068f 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 	
     #@defines
-	PREFIX = /usr
+	PREFIX ?= /usr
 	BIN_DIR = $(PREFIX)/bin
 	SHARE_DIR = $(PREFIX)/share
 	DESKAPPS_DIR = $(SHARE_DIR)/applications
-- 
2.5.5

