From 682ccb0714a60c960b960641e21463f57c7e26f9 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sat, 5 Mar 2016 21:49:22 +0100
Subject: [PATCH] fix system libraries

---
 makefile.linux | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/makefile.linux b/makefile.linux
index 76c9d14..f5bb86c 100644
--- a/makefile.linux
+++ b/makefile.linux
@@ -1,12 +1,8 @@
 CXX = g++
 
-LIBS = src/extralibs/tinyxml/tinyxml.o \
-       src/extralibs/tinyxml/tinyxmlparser.o \
-       src/extralibs/tinyxml/tinystr.o \
-       src/extralibs/tinyxml/tinyxmlerror.o \
-       -lasound -lSDL -lsdldraw -lz -ljack -lpthread
+LIBS = -lasound -lSDL -lsdldraw -lz -ljack -lpthread -ltinyxml
 
-FLAGS = -g -O3 -fno-strict-aliasing -ffast-math -fno-rtti -fexceptions -Wno-write-strings -Wno-multichar -Lsrc/extralibs/sdl_draw -Lsrc/extralibs/zlib-1.2.3 \
+FLAGS = -g -O3 -fno-strict-aliasing -ffast-math -fno-rtti -fexceptions -Wno-write-strings -Wno-multichar -Lsrc/extralibs/sdl_draw \
         -D __LINUX__ -D __GCC__ -D __LINUX_ALSASEQ__ -D __MOT_SWAP__
 
 EXEOBJS = ptk.o \
@@ -32,8 +28,6 @@ RemObj:
 
 protrekkr: $(EXEOBJS)
 	(cd src/extralibs/sdl_draw; make -f makefile.linux)
-	(cd src/extralibs/tinyxml; make)
-	(cd src/extralibs/zlib-1.2.3; make)
 	$(CXX) -o ./release/distrib/ptk_linux -L./ $(FLAGS) $(EXEOBJS) $(LIBS)
 	strip -s -x ./release/distrib/ptk_linux
 #	-rm -f $(EXEOBJS)
-- 
2.5.0

